goreleaser.yml

```diff
@@ -21,6 +22,24 @@ builds:
       - -X 'github.com/ZupIT/horusec/cmd/app/version.Version={{ .Env.CLI_VERSION }}'
       - -X 'github.com/ZupIT/horusec/cmd/app/version.Commit={{ .Commit }}'
       - -X 'github.com/ZupIT/horusec/cmd/app/version.Date={{ .Env.CURRENT_DATE }}'
+      - -s -w
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+      - 386
+  - id: horusec-standalone
+    env: [ CGO_ENABLED=0 ]
+    main: ./cmd/app/main.go
```

Comment:
I think that would be better remove `main.go` and let just `./cmd/app/`, with this any new file on cmd/app will be compiled automatically.

Reply:
the goreleaser file needs a main func for reference, if we need to build another binary we can create a new id on builds section pointing to the new main file